### PR TITLE
js/bundle: Add tests

### PR DESCRIPTION
**Pull request by irori**
Also bumps package version to 0.0.2.

### CODE REVIEW

File: js/bundle/tests/decoder_test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const wbn = require('../lib/wbn');
+
+describe('Bundle', () => {
+  const bundleBuffer = (() => {
```
Comment (horo-t) [inline js/bundle/tests/decoder_test.js]: I think we should have a pre-generated test bundle file to detect unexpected regressions (or format change).

File: js/bundle/tests/decoder_test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    const builder = new wbn.BundleBuilder('https://example.com/');
+    builder.setManifestURL('https://example.com/manifest.json');
```
Comment (horo-t) [inline js/bundle/tests/decoder_test.js]: We should have no-manifest wbn test case.

File: js/bundle/tests/decoder_test.js (hunk 1 above)
Comment (irori) [reply js/bundle/tests/decoder_test.js]: Done.

File: js/bundle/tests/decoder_test.js (hunk 2 above)
Comment (irori) [reply js/bundle/tests/decoder_test.js]: The pre-generated wbn doesn't have a manifest.

**Pull request merged**